auth0-express-js-sample/client/src/components/BootstrapNavbar.js

```diff
@@ -24,7 +18,8 @@ class BootstrapNavbar extends React.Component {
                             <Navbar.Toggle aria-controls="basic-navbar-nav" />
                             <Navbar.Collapse id="basic-navbar-nav">
                                 <Nav className="mr-auto">
-                                    <Interactive className="ml-auto" as={HashLink} smooth className="nav-link"
+                                    {/* <Interactive className="ml-auto" as={HashLink} smooth className="nav-link" */}
+                                    <Interactive as={HashLink} smooth className="nav-link"
```

Comment:
the `<Interactive ...` received `className` 2 times (`ml-auto` and `nav-link`) which is impossible.
So I removed the `ml-auto` as it matched the rest. Make sure this is what you need